openapi-security/src/main/java/com/networknt/openapi/SimpleJwtVerifyHandler.java

```diff
@@ -0,0 +1,268 @@
+                        logger.trace("claims = {}", claims.toJson());
+
+                    /* if no auditInfo has been set previously, we populate here */
+                    auditInfo = exchange.getAttachment(AttachmentConstants.AUDIT_INFO);
+                    if (auditInfo == null) {
+                        auditInfo = new HashMap<>();
+                        exchange.putAttachment(AttachmentConstants.AUDIT_INFO, auditInfo);
+                    }
+
+                    String clientId = claims.getStringClaimValue(Constants.CLIENT_ID_STRING);
+                    String userId = claims.getStringClaimValue(Constants.USER_ID_STRING);
+                    String issuer = claims.getStringClaimValue(Constants.ISS_STRING);
+                    // try to get the cid as some OAuth tokens name it as cid like Okta.
+                    if (clientId == null)
+                        clientId = claims.getStringClaimValue(Constants.CID_STRING);
+
+
+                    // try to get the uid as some OAuth tokens name it as uid like Okta.
+                    if (userId == null)
+                        userId = claims.getStringClaimValue(Constants.UID_STRING);
+
+                    auditInfo.put(Constants.USER_ID_STRING, userId);
+                    auditInfo.put(Constants.SUBJECT_CLAIMS, claims);
+                    auditInfo.put(Constants.CLIENT_ID_STRING, clientId);
+                    auditInfo.put(Constants.ISSUER_CLAIMS, issuer);
+
+                    if (!config.isEnableH2c() && checkForH2CRequest(headerMap)) {
+                        setExchangeStatus(exchange, STATUS_METHOD_NOT_ALLOWED);
+                        if (logger.isDebugEnabled()) logger.debug("SimpleJwtVerifyHandler.handleRequest ends with an error.");
+                        return false;
+                    }
+
+                    String callerId = headerMap.getFirst(HttpStringConstants.CALLER_ID);
+
+                    if (callerId != null)
+                        auditInfo.put(Constants.CALLER_ID_STRING, callerId);
+
+                    // pass through claims through request headers after verification is done.
+                    if(config.getPassThroughClaims() != null && config.getPassThroughClaims().size() > 0) {
+                        for(Map.Entry<String, String> entry: config.getPassThroughClaims().entrySet()) {
+                            String key = entry.getKey();
+                            String header = entry.getValue();
+                            Object value = claims.getClaimValue(key);
+                            if(logger.isTraceEnabled()) logger.trace("pass through header {} with value {}", header, value);
+                            headerMap.put(new HttpString(header), value.toString());
+                        }
+                    }
+                    if (logger.isTraceEnabled())
+                        logger.trace("complete JWT verification for request path = {}", exchange.getRequestURI());
```

Comment:
For consistency let's replace `JWT` for `SJWT`

Reply:
Thanks. It is updated.